experiment/logviewer/gcsreader/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"io"
+	"os"
+	"regexp"
+
+	"cloud.google.com/go/storage"
+	gzip "github.com/klauspost/pgzip"
+	"github.com/kubernetes/klog"
+	"google.golang.org/api/option"
+)
+
+func main() {
+	run()
+}
+
+func run() {
```

Comment:
Only main should call klog.Fatal. Please either merge this with main (preferably) or change 'run' to return error and not call klog.Fatal there :)

Reply:
The reason why i kept this method is that you can do local run just by replacing 1 line of code in main instead of commenting/deleting whole 'main'

Reply:
Looks convenient for me, but as I see, its a bit confusing, so I will inline it into main :)